src/modules/InstURL.rb

```diff
@@ -142,232 +83,45 @@ def RewriteCDUrl(url)
     # Convert install.inf to a URL useable by the package manager
     # @param [String] extra_dir append path to original URL
     # @return [String] new repository URL
-    def installInf2Url(extra_dir)
-      repo_url = ""
-
-      # bnc #406162
-      repo_url_from_inf = false
-      Builtins.foreach(["ZyppRepoURL"]) do |in_inf_key|
-        repo_url = Linuxrc.InstallInf(in_inf_key)
-        if repo_url != nil && repo_url != ""
-          Builtins.y2milestone(
-            "Using %1 directly from install.inf: %2",
-            in_inf_key,
-            URL.HidePassword(repo_url)
-          )
-          repo_url_from_inf = true
-          raise Break
-        end
-      end
+    def installInf2Url(extra_dir = "")
+      return @installInf2Url unless @installInf2Url.nil?
 
-      return repo_url if repo_url_from_inf
+      parts = URL.Parse(Linuxrc.InstallInf("ZyppRepoURL"))
```

Comment:
I wanted to check the documentation of install.inf contents, but I could not find it at http://www.rubydoc.info/github/yast/yast-yast2/Yast/LinuxrcClass#InstallInf-instance_method . It should link to the following, I guess:

I did find https://en.opensuse.org/SDB:Linuxrc_install.inf but it does not document `ZyppRepoURL`, only

> xxxZyppRepoURL 	String 	Not in use yet. After renaming to ZyppRepoURL will contain libzypp installation repository URL. 

@wfeldt Steffen, is there a place documenting this which I overlooked?